website/docs/r/logging_project_view.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+subcategory: "Cloud (Stackdriver) Logging"
+layout: "google"
+page_title: "Google: google_logging_view"
+sidebar_current: "docs-google-logging-view"
+description: |-
+  Manages a log bucket view.
+---
+
+# google\_logging\_view
+
+Manages a log bucket view. For more information see
+[the official logging documentation](https://cloud.google.com/logging/docs/) and
+[Managing log views on your log buckets](https://cloud.google.com/logging/docs/logs-views).
+
+~> **Note:** Log views are automatically created for a log bucket. Creating a resource of this type will fail in Terraform. The log views that are currently automatically created are `_AllLogs` and `_Default`.
```

Comment:
I think later we can add a datasource for that if will be needed ever. The problem with `_AllLogs` and `_Default` views is that they are not modifiable, which kinda breaks the whole idea of a Terraform-managed resource.